src/packages/verticalsteps/doc.en-US.md

```
@@ -0,0 +1,98 @@
+| --- | --- | --- | --- |
+| title | Title of the process step | `string` | `-` |
+| description | Descriptive text of process steps (supporting HTML structure) | `string` | `-` |
+| icon | Icon | `ReactNode` | `-` |
+| value | Index of process steps | `number` | `0` |
+| onStepClick | Fired when the title or icon of a step is clicked | `(index: number) => void` | `-` |
+
+## Theming
+
+### CSS Variables
+
+The component provides the following CSS variables, which can be used to customize styles. Please refer to [ConfigProvider component](#/en-US/component/configprovider).
```

Markdownlint

Link fragments should be valid

(MD051, link-fragments)